purchase_product_usage/models/account_move.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Aleph Objects, Inc.
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0).
+
+from odoo import api, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    @api.onchange(
+        "amount_currency",
+        "currency_id",
+        "debit",
+        "credit",
+        "tax_ids",
+        "account_id",
+        "analytic_account_id",
+        "analytic_tag_ids",
+    )
+    def _onchange_mark_recompute_taxes(self):
+        for line in self:
+            if line.purchase_line_id.usage_id.account_id:
+                account = line.purchase_line_id.usage_id.account_id
+            else:
+                account = line._get_computed_account()
+            line.account_id = account
```

Comment:
Apply https://github.com/OCA/purchase-workflow/pull/1111

Reply:
I have changed the code as you mentioned. Thank you for your advice.

Reply:
@flook-chaiwat as a good practice. You should use `git cherry-pick`, instead of copy code to fix. We always try to keep the original author in the commit history.